tools/deploy_translatable_strings.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Script for pushing the translatable messages to poBranch.
+
+
+# Variables used by this script.
+# From github actions the docs/locale/en directory from the sphinx build
+# gets downloaded from the github actions artifacts as deploy directory
+TARGET_REPOSITORY="git@github.com:qiskit-community/qiskit-translations.git"
+SOURCE_DIR=`pwd`
+SOURCE_LANG='en'
+
+SOURCE_REPOSITORY="git@github.com:Qiskit/qiskit.git"
+TARGET_BRANCH_PO="master"
```

Comment:
```suggestion
TARGET_BRANCH_PO="main"
```

The script has been broken for 2+ months because of this, which is a large part of why I rewrote it (and to better integrate with the actions dependency graph).